<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LT-MRADFORD01" ignored="0" name="Release check" tests="8" failures="2" timestamp="2021-02-08T16:24:52 PST" time="73.462" errors="0">
  <testcase name="checkBoostPages" time="12.235" classname="automationFramework.TSVerifyPages"/>
  <testcase name="checkBoostPages" time="6.756" classname="automationFramework.TSVerifyPages"/>
  <testcase name="checkBoostPages" time="6.247" classname="automationFramework.TSVerifyPages"/>
  <testcase name="checkBoostPages" time="6.504" classname="automationFramework.TSVerifyPages"/>
  <testcase name="checkBoostPages" time="9.793" classname="automationFramework.TSVerifyPages"/>
  <testcase name="checkBoostPages" time="8.889" classname="automationFramework.TSVerifyPages"/>
  <testcase name="checkCategoryPages" time="8.257" classname="automationFramework.TSVerifyPages">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at automationFramework.TSVerifyPages.checkCategoryPages(TSVerifyPages.java:57)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- checkCategoryPages -->
  <testcase name="checkCategoryPages" time="8.614" classname="automationFramework.TSVerifyPages">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at appModule.Listeners.onTestFailure(Listeners.java:51)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- checkCategoryPages -->
</testsuite> <!-- Release check -->
